dev-workflow-ui/webContent/includes/template/notifications.xhtml

```diff
@@ -37,11 +37,18 @@
         <p:dataScroller id="notifications" lazy="true" styleClass="notification-scroll" value="#{notificationBean.dataModel}" var="notification" chunkSize="20" mode="inline">
           <div class="#{notification.style} notification-container">
             <div class="p-d-flex p-jc-between">
-              <h:outputText id="notificationMessage" value="#{notification.message}" escape="false"/>
+              <h:outputLink id="notificationMessage" value="#{notification.infoAction.link()}" styleClass="notification-message">
+                <h:outputText value="#{notification.message}"/>
+              </h:outputLink>
+              <p:tooltip for="notificationMessage" value="Info" />
 
-              <div class="p-d-flex">
+              <div class="p-d-flex notification-action-container">
+                <h:outputLink id="notificationRunAction" value="#{notification.runAction.link()}" rendered="#{notification.runAction != null}">
+                  <i class="notification-action si si-controls-play p-mr-1 " />
```

Comment:
get the css icon from the action itself. The action should be a DTO that delivers different CSS icons for different kind of actions